modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/expression/DDMFormEvaluatorExpressionActionHandler.java

```diff
@@ -50,6 +50,25 @@ public ExecuteActionResponse executeAction(
 		return builder.build();
 	}
 
+	protected boolean hasIntervalOnPageFlow(
+		Integer fromPageIndex, Integer toPageIndex) {
+
+		for (Map.Entry<Integer, Integer> entry : _pageFlow.entrySet()) {
+			int fromPageFlowIndex = entry.getKey();
+			int toPageFlowIndex = entry.getValue();
+
+			if ((toPageIndex < fromPageFlowIndex) ||
+				(fromPageIndex > toPageFlowIndex)) {
+
+				continue;
+			}
+
+			return true;
+		}
+
+		return false;
+	}
+
```

Comment:
Hi @linolaoj,
The code looks great to me, but I'm not sure I really understand what it's doing. Could you explain to me? Thanks

Reply:
Hi @carolmariaabb ,
on https://github.com/liferay/liferay-portal/blob/a4e6ca098566e12cd033072c7e5326a3bbe5ae8b/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java#L448 the pages inside the interval of the "Jump to page" are disabled. What this change does is not allowing to include a "Jump to page" of a page that will be disabled. To check that, the method `hasIntervalOnPageFlow` checks if the "Jump to page" happens before or after an already existing "Jump to page".
Please let me know if more details are needed.